src/OrchardCore.Modules/OrchardCore.Admin/Admin/AdminSettings.cs

```diff
@@ -3,5 +3,7 @@ namespace OrchardCore.Admin.Models
     public class AdminSettings
     {
         public bool DisplayMenuFilter { get; set; }
+
+        public bool DisplayDarkmode { get; set; }
```

Comment:
I think should be true by default.

Otherwise people will not know it's available

(Same with `DisplayMenuFilter` IMO @agriffard do you agree / disagree?)